Dockerfile.template.erb

```diff
@@ -159,35 +153,33 @@ RUN mkdir -p /fluentd/log
 # configuration/plugins path (default: copied from .)
 RUN mkdir -p /fluentd/etc /fluentd/plugins
 
+<% if is_alpine %>
+RUN addgroup -S fluent && adduser -S -g fluent fluent
+<% else %>
+RUN groupadd -r fluent && useradd -r -g fluent fluent
+<% end %>
+RUN chown -R fluent /fluentd && chgrp -R fluent /fluentd
+
 COPY fluent.conf /fluentd/etc/
 COPY entrypoint.sh /bin/
 RUN chmod +x /bin/entrypoint.sh
 <% end %>
 
-<% if is_onbuild %>
-ONBUILD COPY fluent.conf /fluentd/etc/
-ONBUILD COPY plugins /fluentd/plugins/
-<% end %>
```

Comment:
For re-introducing the `ONBUILD` images, I'd recommend using a separate simpler `Dockerfile`, something like:

```console
FROM fluentd:x.y.z-debian
ONBUILD COPY fluent.conf /fluentd/etc/
ONBUILD COPY plugins /fluentd/plugins/
```

(Which both serves to ease maintenance, and makes it really obvious to users who want to move away from the `onbuild` variant what they need to do to replicate the behavior that was provided for them.)

Reply:
I can look into this.